rules/data/embedded/embedded-base64-gzip.yara

```diff
@@ -28,12 +28,15 @@ rule base64_obfuscated_js: critical {
     description = "Contains base64 gzip content within high-entropy javascript"
 
   strings:
-    $header = "H4sIA"
-    $       = "charAt("
-    $       = "substr("
-    $       = "join("
-    $       = "function("
+    $f_header          = "H4sIA"
+    $f_char_at         = "charAt("
+    $f_substr          = "substr("
+    $f_join            = "join("
+    $f_function        = "function("
+    $not_events_js_map = "//# sourceMappingURL=events.js.map"
+    $not_jitsu1        = "admin-service-account@jitsu.com"
+    $not_jitsu2        = "jitsu-auth"
 
   condition:
-    filesize < 2MB and all of them and math.entropy(1, filesize) >= 5.0
+    filesize < 2MB and all of ($f*) and math.entropy(1, filesize) >= 5.0 and none of ($not*)
```

Comment:
mark this rule as high

Reply:
Updated in [`4260efd` (#694)](https://github.com/chainguard-dev/malcontent/pull/694/commits/4260efda23707a4e7d4a8539902b41a090f16684).